modules/travel_pay/app/services/travel_pay/claims_client.rb

```diff
@@ -23,5 +23,35 @@ def get_claims(veis_token, btsss_token)
         req.headers.merge!(claim_headers)
       end
     end
+
+    ##
+    # HTTP POST call to the BTSSS 'claims' endpoint
+    # API responds with a new travel pay claim ID
+    #
+    # @params {
+    #  "appointmentId": "string", (BTSSS internal appointment ID - uuid)
+    #  "claimName": "string", (internal to BTSSS)
+    #  "claimantType": "Veteran" (currently, "Veteran" is the only claimant type supported)
+    # }
+    #
+    # @return claimID => string
+    #
+    def create_claim(veis_token, btsss_token, params = {})
+      btsss_url = Settings.travel_pay.base_url
+      correlation_id = SecureRandom.uuid
+      Rails.logger.debug(message: 'Correlation ID', correlation_id:)
+
+      connection(server_url: btsss_url).post('api/v1.1/claims') do |req|
```

Comment:
I wondered if we wanted to create a variable for the version of the external API we're using. However, the new `POST` to the `/claims` endpoint is only available on v1.1, but that isn't in prod yet, so the "get claims" would fail, but I feel like we should be consistent once v1.1 is in prod.... 🤔

Reply:
@liztownd When will the v1.1 be deployed to prod?

Reply:
Yeah, correct @liztownd. @RachalCassity this is something we're working with our partners on. It kind of is what it is right now, but we're hoping to get them aligned to code.va.gov API standards sooner rather than later.